ui/effect-blind.js

```diff
@@ -12,6 +12,7 @@
 //>>description: Blinds the element.
 //>>docs: http://api.jqueryui.com/blind-effect/
 //>>demos: http://jqueryui.com/effect/
+//>>css.theme: ../themes/base/theme.css
```

Comment:
Again, I don't think any of the effects have any CSS dependency at all. Going to skip the other `effect*` files.